build.rs

```diff
@@ -1,3 +1,9 @@
 fn main() {
+    // Support for fixed-address compilation.
     libtock_build_scripts::auto_layout();
+
+    // Configure the alignment size for the linker. This prevents the linker
+    // from assuming very large pages (i.e. 65536 bytes) and unnecessarily
+    // inserting additional padding into the output ELF.
+    println!("cargo:rustc-link-arg=-zmax-page-size=4096");
```

Comment:
Any reason not to include this in `libtock_build_scripts::auto_layout`?

Reply:
My thinking was users may want this but not auto_layout.

We could add it to the build-scripts crate.

Reply:
I feel think this is part of configuring the layout for a `libtock-rs` app, and therefore belongs in `auto_layout`.